src/main/scala/org/zalando/spearheads/innkeeper/utils/EnvConfig.scala

```diff
@@ -0,0 +1,52 @@
+package org.zalando.spearheads.innkeeper.utils
+
+import com.google.inject.Inject;
+import com.typesafe.config.Config
+import scala.collection.JavaConversions._
+import scala.util.Try
+
+/**
+ * Tries to get a config depedent on the current environment.
+ *
+ * If it can't find it in the current environment, it will fallback to the default config
+ *
+ */
+trait EnvConfig {
```

Comment:
Not for this PR, but as a general suggestion: In another large scala project, we decided have the config explicitly typed and containing functions. Something like
```
trait EnvConfig {
  def getOauthScopeRead: Set[String]
}
```
The idea is to avoid runtime failures just because someone has a typo in a key string. Also makes it very easy (and compile time checked) which configs are no longer used. Internally you still load them with `config.getString(...)` but the actual string is used just once in the whole code base inside of the private EnvConfig implementation. Every usage of that config value is via explicit, named function calls as outlined above.